### PR TITLE
login api added as per requirements #2

### LINKED ISSUE #2: Login API endpoint
payload:
```
{
  username: {String},
  password: {String}
}
```

On success, it should return a JWT

### PR DESCRIPTION
**Pull request by sankethkini**
@isuruAb  I have added API for login which in return gives jsonwebtoken for users. 

Fixes: #2

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,13 +1,45 @@
...
+})
+
+//api routes
+const route=require('./Routes/userRoutes');
+app.use("/api",route);
+
+//exception for wrong requests
+app.use((req,res,next)=>{
+  var error=new HttpError('not found',404);
+  next(error);
```
Comment (isuruAb) [inline app.js:34]: nice

File: Routes/userRoutes.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const Route=express.Router();
+const User=require('../Model/User');
+const HttpError=require('../Model/HttpError');
```
Comment (isuruAb) [inline Routes/userRoutes.js]: Do we need this here?

File: Controller/loginController.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                        return next(new HttpError('tokenization error',500));
+                    }
+                    else{
+                        return res.status(200).json({"token":token});
+                    }
+                })
+            }
+        }
+    })
+    // return res.send('success');
```
Comment (isuruAb) [inline Controller/loginController.js]: Please remove this comment

**Review by isuruAb — CHANGES_REQUESTED** [review]
Good work. Please check the inline comments.

**Comment by sankethkini** [comment]
@isuruAb  Should I need to remove inline comments or make them more meaningful?

**Comment by isuruAb** [comment]
> @isuruAb Should I need to remove inline comments or make them more meaningful?

I don't think it is necessary. you have done a wonderful job there 👍🏽

**Review by isuruAb — APPROVED** [review]
LGTM. Thank you for your contribution @sankethsolo  👍🏽

**Pull request merged**